provider-ui/pages/auth/login.js

```diff
@@ -5,15 +5,15 @@ import Dialog from '@material-ui/core/Dialog';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogContentText from '@material-ui/core/DialogContentText';
 import DialogTitle from '@material-ui/core/DialogTitle'
-import InfoIcon from '@material-ui/icons/Info';
+import WarningIcon from '@material-ui/icons/Warning';
```

Comment:
Revert login expire modal change, not related to this pr, I see you have another one opened for that.

Reply:
@legendarykamal Please resolve this review also, we are good to go then.

Reply:
> Revert login expire modal change, not related to this pr, I see you have another one opened for that.

@Chadha93 this issue is related to?

Reply:
@Chadha93 isn't it already reverting to prev state with the handleclose function?

Reply:
Please read the review comment again. :) and see https://github.com/meshery/meshery/pull/5842

Reply:
```
  if (countDown === 1) {
    handleClose();
    window.location = "/user/login";
  }
```
this already reverts to the login page  , m i missing something @Chadha93 ?

Reply:
Okay, I'll rephrase - by reverting the change I meant remove this particular file changes form this pr(not related issue here) as it's conflicting with the changes from your other pr. You have another pr for this particular change. Please visit your another pr #5835 which also contains these same changes.

Reply:
thanks i got it. @Chadha93  I have made the changes and hope it's good to merge now :)